requirements-docs.txt

```diff
@@ -0,0 +1,8 @@
+-rrequirements.txt
+sphinx
+rinohtype
```

Comment:
?

Reply:
I removed this package from requirements